jest/jest-setup.ts

```diff
@@ -1,62 +1,8 @@
-import '@testing-library/jest-dom/extend-expect'
-import '@testing-library/jest-dom'
-
-// Polyfills `window.fetch` for Jest because it runs in a Node environment where fetch isn’t available. It initially looked like this would change with Node.js 18, but that is not so.
-import 'isomorphic-fetch'
-
 import dotenv from 'dotenv'
 
-import { replaceAttributesSnapshotSerializer } from './jest-replace-attribute-snapshot-serializer'
-import { setupMockServer } from '../src/api/setupMockServer'
-
 /**
  * Ensures the tests run with the project’s environment variables being available.
- */
-dotenv.config()
-
-/**
- * amcharts need SVGPathElement defined for our tests to work.
  *
- * See: https://github.com/amcharts/amcharts4/issues/1387
+ * It’s important to run the environment variable setup as part of a Jest setup file and before a Jest setup after env file because the latter might statically import a file that makes use of environment variables which wouldn’t be correctly resolved at the time the static imports executed.
  */
-Object.defineProperty(window, 'SVGPathElement', { value: class extends HTMLElement { } })
-
-/**
- * Kongponents v6 uses a call to `crypto.getRandomValues` which isn’t implemented in jsdom by default.
- * To make the tests run, we supply it via Node’s `crypto` library.
- *
- * See: https://github.com/jsdom/jsdom/issues/1612
- */
-Object.defineProperty(global.self, 'crypto', {
```

Comment:
I manually diffed the `jest-setup.ts` and `jest-setup-after-env.ts` - and I think we're missing this. Do we not need it any more or is this a bug?

Reply:
Yep. I checked whether some of the things in here were still necessary and this wasn’t.